sdk/evaluation/azure-ai-evaluation/tests/e2etests/test_evaluate.py

```diff
@@ -29,6 +29,12 @@ def data_file():
     return os.path.join(data_path, "evaluate_test_data.jsonl")
 
 
+@pytest.fixture
+def data_file_no_query():
+    data_path = os.path.join(pathlib.Path(__file__).parent.resolve(), "data")
+    return os.path.join(data_path, "evaluate_test_data_no_query.jsonl")
```

Comment:
I don't see this file in data folder. Can you check?

Reply:
Just pushed it - good catch. It was untracked in git.